include/fizzy/fizzy.h

```diff
@@ -596,18 +596,31 @@ bool fizzy_find_exported_memory(
 bool fizzy_find_exported_global(
     FizzyInstance* instance, const char* name, FizzyExternalGlobal* out_global) FIZZY_NOEXCEPT;
 
+FizzyExecutionContext* fizzy_create_execution_context(int depth) FIZZY_NOEXCEPT;
+
+FizzyExecutionContext* fizzy_create_metered_execution_context(
+    int depth, int64_t ticks) FIZZY_NOEXCEPT;
+
+void fizzy_free_execution_context(FizzyExecutionContext* ctx) FIZZY_NOEXCEPT;
+
+int* fizzy_get_execution_context_depth(FizzyExecutionContext* ctx) FIZZY_NOEXCEPT;
```

Comment:
Does this returning `int*` look fine, or would `_get`/`_set` methods be better?

Reply:
Not sure, need to try in Rust how to interact with that, but we could merge this as a first step anyway.